tools/scripts/update_assets_metadata.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+import json
+import hashlib
+import re
+
+ASSETS_JSON = './rotkehlchen/data/all_assets.json'
+ASSETS_META = './rotkehlchen/data/all_assets.meta'
+ASSETS_TEST = './rotkehlchen/tests/unit/test_assets.py'
```

Comment:
I guess this only works if you run the script from the root directory. We can change that to be relevant to this file, so it can work no matter where it's called from.